OpenShift-Installation/eVDF-Openshift-Installation.rst

```diff
@@ -0,0 +1,592 @@
+
+5. Install the eVDF bootstrap agent “Nuage-evdf-x.y.z-b-bundle.bin” on the node with “-d” option. (where x.y.z-b represents your Nuage version)
+
+   ::
+    
+      ./Nuage-evdf-x.y.z-b-bundle.bin -d
+
+ 
+
+Installation for a Single Master
+-----------------------------------
+
+1. Create a nodes file for Ansible configuration for a single master in the openshift-ansible directory with the contents shown below.
+
+2. Verify that the image versions are accurate by checking the TAG displayed by 'docker images' output for successful deployment of Nuage daemonsets: 
+
+  .. Note:: The following nodes file is provided as a sample. Please update the values with your actual deployment. The below nodes file deploys OpenShift version 3.7.9. To deploy OpenShift version 3.6, use 'openshift_pkg_version=-3.6.173.0.5' or to deploy OpenShift version 3.5, use 'openshift_pkg_version=-3.5.5.5'
+  
+::
+
+    # Create an OSEv3 group that contains the masters and nodes groups
+    [OSEv3:children]
+    masters
+    nodes
+    etcd 
+    
+    # Set variables common for all OSEv3 hosts
+    [OSEv3:vars]
+    # SSH user, this user should allow ssh based auth without requiring a password
+    ansible_ssh_user=root
+    openshift_master_portal_net=172.30.0.0/16
+    osm_cluster_network_cidr=70.70.0.0/16
+    deployment_type=openshift-enterprise
+    osm_host_subnet_length=10
+    openshift_pkg_version=-3.7.9
+
+    # If ansible_ssh_user is not root, ansible_sudo must be set to true
+    #ansible_sudo=true 
+    
+    deployment_type=openshift-enterprise 
+    openshift_disable_check=disk_availability,memory_availability,docker_storage,docker_image_availability,package_version,package_availability
+    
+    # Nuage specific parameters
+    openshift_use_openshift_sdn=False
+    openshift_use_nuage=True
+    openshift.common._use_nuage=True
+    os_sdn_network_plugin_name=cni
+    vsd_api_url=https://<VSD-IP/VSD-Hostname>:7443
+    vsp_version=v5_0
+    
+    # The below versions should match the TAG version in the output of 'docker images' on the nodes. See point 2 above
+    # Example: nuage_monitor_image_version=5.2.2-70
+    nuage_monitor_image_version=<version>
+    nuage_vrs_image_version=<version>
+    nuage_cni_image_version=<version>
+    nuage_infra_image_version=<version>
+    nuage_strongswan_image_version=<version>
+    enterprise=openshift
+    domain=openshift
+    vsc_active_ip=10.100.100.101
+    vsc_standby_ip=10.100.100.102
+    nuage_personality=evdf
+    nw_uplink_intf=eth0
+    evdf_uplink_intf=eth0
+    nuage_site_id=-1
+    enable_underlay_support=1
+    enable_stats_logging=1
+    vrs_bridge_mtu_config=1450
+    nuage_interface_mtu=1350
+    nuage_openshift_monitor_log_dir=/var/log/nuage-openshift-monitor
+    # auto scale subnets feature
+    # 0 => disabled(default)
+    # 1 => enabled
+    auto_scale_subnets=0
```

Comment:
Except for evdf_uplink_intf and personality, is there anything else that needs to be set?